scripts/gpg_add_key_to_keyring.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
```

Comment:
Alll these scripts are missing a copyright header or at least a copyright note.
If you want to publish this as "public domain", I suggest you to choose the CC0 Licend and add a short note like this:
```
Licensed under Creative Commons CC0 1.0 Universal - No Rights Reserved / Public Domain
```

Reply:
I would rather prefer 

```
/*
* ----------------------------------------------------------------------------
* "THE BEER-WARE LICENSE" (Revision 42):
* <phk@FreeBSD.ORG> wrote this file. As long as you retain this notice you
* can do whatever you want with this stuff. If we meet some day, and you think
* this stuff is worth it, you can buy me a beer in return Poul-Henning Kamp
* ----------------------------------------------------------------------------
*/
```

Reply:
@htgoebel The whole repository has a [MIT License](https://github.com/debops/examples/blob/master/LICENSE), I believe that if scripts or other files don't have a separate license, they will fall under that one, correct?

Reply:
That's correct, either one "global" license per repository or one license per file/role/etc.